### ISSUE TITLE
Django 4.1- ckeditor widget not appearing in admin

### ISSUE THREAD

**Issue by magnus-haw** [django-ckeditor/django-ckeditor#728]
The widget is not appearing the the Django admin. There is also this line (/static/ckeditor/ckeditor-init.js" data-ckeditor-basepath="/static/ckeditor/ckeditor/" id="ckeditor-init-script) which appears at the top of the DOM body and shows up above the admin bar. Presumably the init script is not being properly activated?

![Screen Shot 2022-09-06 at 5 43 27 PM](https://user-images.githubusercontent.com/26490985/188764493-61475c4c-34a2-4953-b6f7-6d01eafd98c6.png)

![Screen Shot 2022-09-06 at 5 43 13 PM](https://user-images.githubusercontent.com/26490985/188764506-c98dc7cb-ea94-477c-ac47-39aed87521a1.png)

**Comment by matthiask** [comment]
Yes, you should probably update django-js-asset; the old version doesn't
work correctly with Django 4.1

**Comment by Vicedominus** [comment]
Hi there budy. Did you try editing admin.py?
This work for me, inside admin.py:

from django.contrib import admin
from .models import *
from django import forms
from ckeditor_uploader.widgets import CKEditorUploadingWidget 

# Register your models here.

class PostAdminForm(forms.ModelForm):
    content = forms.CharField(widget=CKEditorUploadingWidget())
    class Meta:
        model = Post
        fields = '__all__'

class PostAdmin(admin.ModelAdmin):
    form = PostAdminForm

admin.site.register(Post, PostAdmin)

**Comment by some1ataplace** [comment]
One workaround for this issue is to manually specify the path to the ckeditor-init.js script in your project's base_site.html template. To do this, first create a templates directory in your project's root directory if it doesn't already exist. Then, create a new file called admin/base_site.html and add the following code:

```
{% extends "admin/base.html" %}

{% block extrahead %}
    {{ block.super }}
    <script src="{% static 'ckeditor/ckeditor/ckeditor.js' %}"></script>
    <script src="{% static 'ckeditor/ckeditor-init.js' %}"></script>
{% endblock %}
```


This will manually load the CKEditor scripts required to display the editor in the Django admin. Make sure to run `python manage.py collectstatic` to collect your static files after making this change.

**Comment by matthiask** [comment]
Fixed in 6.5.1.
